GoogleForCreators/web-stories-wp
# Redesign: Page background

## Summary
Redesigns Page background panel. Also removes Size & Position for background media as a separate panel and moves it under Page background.


## Relevant Technical Choices



## To-do




## User-facing changes
| Before | After |
|-------|-------|
| <img width="296" alt="Screenshot 2021-03-03 at 18 30 22" src="https://user-images.githubusercontent.com/3294597/109838119-2a0d4600-7c46-11eb-9356-f8b726b592fd.png"> | <img width="314" alt="Screenshot 2021-03-03 at 18 31 33" src="https://user-images.githubusercontent.com/3294597/109838327-57f28a80-7c46-11eb-9a87-17099706d535.png"> |




## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Verify that Page background panel looks as in Figma when Page is selected
2. Add media and set is as background
3. Verify that the Page Background panel includes Media removal and flip controls and these work as expected.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6461
